kartothek/io/dask/bag.py

```diff
@@ -288,3 +292,25 @@ def build_dataset_indices__bag(
             update_indices_from_partitions, dataset_metadata_factory=ds_factory
         )
     )
+
+
+def collect_dataset_statistics(store, dataset_uuid, table_name, predicates=None, sample_size=0.3):
```

Comment:
docsting

Reply:
I would call the `sample_size` `frac` leaning onto https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sample.html